PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -134,11 +159,14 @@
                 " bit 10  for hltL1sQuadJetC60IorHTT380IorHTT280QuadJetIorHTT300QuadJet or hltL1sQuadJetC50to60IorHTT280to500IorHTT250to340QuadJet" \
                 " bit 11 for hltBTagCaloCSVp05Double or hltBTagCaloDeepCSVp17Double, bit 12 for hltPFCentralJetLooseIDQuad30, bit 13 for hlt1PFCentralJetLooseID75," \
                 " bit 14 for hlt2PFCentralJetLooseID60, bit 15 for hlt3PFCentralJetLooseID45, bit 16 for hlt4PFCentralJetLooseID40," \
-                " bit 17 for hltBTagPFCSVp070Triple or hltBTagPFDeepCSVp24Triple or hltBTagPFDeepCSV4p5Triple "),
+                " bit 17 for hltBTagPFCSVp070Triple or hltBTagPFDeepCSVp24Triple or hltBTagPFDeepCSV4p5Triple,"\
+                " bit 18 for Double tau + jet,"\
+                " bit 19 for VBF cross-cleaned from medium deeptau PFTau,"+
+                " bit 20 for VBF cross-cleaned using dijet correlation checker ")
         ),
         cms.PSet(
             name = cms.string("FatJet"),
-            id = cms.int32(6),
+            id = cms.int32(7),
```

Comment:
for back-compatibility, don't change `FatJet` id, just use some new id for `BoostedTau`. E.g. I propose `1515` :)